fitlins/interfaces/nistats.py

```diff
@@ -90,6 +69,15 @@ def _run_interface(self, runtime):
         mat.to_csv('design.tsv', sep='\t')
         self._results['design_matrix'] = os.path.join(runtime.cwd,
                                                       'design.tsv')
+        return runtime
+
+
+class FirstLevelModel(NistatsBaseInterface, FirstLevelEstimatorInterface, SimpleInterface):
```

Comment:
should the name of this class remain `FirstLevelModel` (and be identifiable by the fact that it lives in nistats.py) or have a more specific name?

Reply:
I see that the `NistatsBaseInterface` also has a `_pkg` variable (does that have some special NiPype meaning?)

Reply:
Yeah, at least for now I'm going to do that. Also other tools aren't obliged to use `FirstLevelModel`, which corresponds to the nistats object.

Reply:
> I see that the `NistatsBaseInterface` also has a `_pkg` variable (does that have some special NiPype meaning?)

Yes, that exists to allow run-time detection of whether a Python module is installed and its version. So if I want to modify my behavior based on changes in the underlying library, that's there.